wlct/tournaments.py

```diff
@@ -1044,7 +1044,7 @@ def process_game(self, game):
                                             if player_to_use.team.id not in teams_lost and len(teams_won) == 0:
                                                 teams_won.append(player_to_use.team.id)
 
-                                        processGameLog += "\n{} failed to join, forcing loss and deleting game ".format(player_to_use.player.name)
+                                        processGameLog += "\n{} failed to join, forcing loss and deleting game ".format(player[0].name)
```

Comment:
What's the reason for changing these? I'd like to keep it consistent. player_to_use instead if we can so that we know this is the only reference to the tournament player...

Reply:
`player_to_use` might not reflect the current player that is actually being checked. This is mainly just an issue for before a game has started. If a player didnt join in time, the clot checks to see if `player[0]` is on vacation, but `player_to_use` could be any player on the team (I think we created this so we have the correct TourneyTeam obj as subs messes things up).

Reply:
Ok cool, sounds good.